tests/test.data-store.js

```diff
@@ -90,6 +93,7 @@ describe('user', function() {
     });
 
     assert.deepEqual(user, {
+      braintreeToken: null,
       signedIn: false,
       email: null,
       payment_methods: [],
```

Comment:
You should add two tests here:
1. receive a braintree token action and make sure it's added to the user object
2. pre-populate a user object, receive a braintree token action, and make sure the old user object was preserved